includes/wc-admin-update-functions.php

```diff
@@ -67,7 +67,7 @@ function wc_admin_update_0230_db_version() {
  * Remove the note unsnoozing scheduled action.
  */
 function wc_admin_update_0251_remove_unsnooze_action() {
-	as_unschedule_action( WC_Admin_Notes::UNSNOOZE_HOOK, null, WC_Admin_Notes::QUEUE_GROUP );
+	as_unschedule_action( WC_Admin_Notes::UNSNOOZE_HOOK, null, 'wc-admin-data' );
```

Comment:
It looks like this should be `wc-admin-notes`:

https://github.com/woocommerce/woocommerce-admin/blob/fe7b6cca7a095d1a39043bd1b38fc055cf0b121d/src/Notes/WC_Admin_Notes.php#L25

Reply:
@jeffstieler Originally it was scheduled under `wc-admin-data`. At some point the group was changed to `wc-admin-notes` which would mean that older installs had two of the unsnooze actions running (totalling 1440 over 30 days). That would have contributed to users complaining about the number. I added a second call to remove from the second group.